### ISSUE TITLE
Split into multiple crates

### ISSUE THREAD

**Issue by DJMcNab** [oli-obk/priroda#40]
At the moment, priroda is two conceptually seperate but entwined parts, namely the web frontend and the additional support code for debugging MIR.

I propose that we split priroda into two crates, namely `priroda` for the backend support code, and `priroda_web` for the web frontend. I'm happy to do the implementation work on this, although I want to confirm that this would be desirable.

In particular, this would also be required for #31, which would add an additional frontend.

**Comment by bjorn3** [comment]
Makes sense. I think it would be nice if multiple frontends could also be used at the same time.

**Comment by oli-obk** [comment]
Yes, that would be great. We can leave all code in this project, and just split it into crates.

**Comment by DJMcNab** [comment]
Yeah, same repository is my intention. Multiple frontends at the same time could make some sense, are you thinking #31 with browser window for the supporting information? Anything else seems likely to get complicated fast.

I suspect we might end up moving directly over to manually creating svg images, just using https://docs.rs/svg/0.9.1/svg/.
